velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -291,5 +291,37 @@ TEST_F(DateTimeFunctionsTest, dateSub) {
   EXPECT_EQ(parseDate("5881580-07-11"), dateSub("1969-12-31", kMin));
 }
 
+TEST_F(DateTimeFunctionsTest, dateDiffDate) {
+  const auto dateDiff = [&](std::optional<int32_t> date1,
+                            std::optional<int32_t> date2) {
+    return evaluateOnce<int32_t, int32_t>(
+        "date_diff(c0, c1)", {date1, date2}, {DATE(), DATE()});
```

Comment:
would be nice to have at least one test case exercising the `datediff()` name, to prevent folks from inadvertently removing the registration in the future.

Reply:
Thanks, Just added.